tests/test_ssim.py

```diff
@@ -2,96 +2,81 @@
 import itertools
 import pytest
 import tensorflow as tf
-
+import numpy as np
+from PIL import Image
```

Comment:
Consider using `from skimage.io import imread` instead of `PIL.Image`. It's slightly faster, reads images directly into np.arrays and already included in testing dependencies.